python/cuml/test/dask/test_random_forest.py

```diff
@@ -143,9 +183,10 @@ def test_rf_regression_dask(partitions_per_worker, cluster):
         X_train_df, y_train_df = dask_utils.persist_across_workers(
             c, [X_train_df, y_train_df], workers=workers)
 
+        X_test_cudf = cudf.DataFrame.from_pandas(pd.DataFrame(X_test))
```

Comment:
this is kind of convoluted to create the cudf dataframe, wouldn't it be better to pass X_test to a device array and then use cudf's from_gpu_matrix?